### PR TITLE
updates to Apache config

### PR DESCRIPTION
**Pull request by kalexand-rh**
@enj, FYI

Follow up to https://github.com/openshift/openshift-docs/pull/12607

@stuartchuan, PTAL?

### CODE REVIEW

**Comment by stuartchuan** [comment]
Why does we change the htpasswd to SSPI connection for Apache example?

**Comment by enj** [comment]
> Why does we change the htpasswd to SSPI connection for Apache example?

Moving away from the "hello world" `htpasswd` example to realistic GSSAPI example.  Our examples should focus on configurations that customers would actually use.  The `htpasswd` config is trivial and fully automateable, and has no real connection with actually getting complex configurations like GSSAPI working correctly.

**Comment by stuartchuan** [comment]
As the GSSAPI still is Technology Preview features, does it reasonable to write here?

**Comment by enj** [comment]
GSSAPI on Linux has been fully supported for years.  Only SSPI on Windows is in preview.

**Comment by enj** [comment]
Note also that these docs refer to the server side configuration (and not client side features) which is the same between GSSAPI and SSPI.  The server side request header code has not changed in years.

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -1118,12 +1118,12 @@ location.
...
 
-. The *mod_auth_form* module is shipped as part of the *mod_session* package that
-is found in the link:https://access.redhat.com/solutions/392003[Optional channel].
+. Obtain the *mod_auth_gssapi* module from the
+link:https://access.redhat.com/solutions/392003[Optional channel].
 Install the following packages:
 +
 ----
-# yum install -y httpd mod_ssl mod_session apr-util-openssl
+# yum install -y httpd mod_ssl mod_session apr-util-openssl mod_auth_gssapi
```
Comment (stuartchuan) [inline install_config/configuring_authentication.adoc]: `httpd` not need here.

Comment (stuartchuan) [reply install_config/configuring_authentication.adoc]: The following ` # systemctl restart httpd` don't need also

File: install_config/configuring_authentication.adoc [hunk 2]
```diff
@@ -1371,19 +1361,29 @@ additional query parameters):
...
+.. Confirm that you can log in to the `oc` command line:
 +
 ----
-#  curl -k -v -u <your_user>:<your_password> \
-    -H 'X-Csrf-Token: 1' '<redirected_challengeURL_from_step_3 +query>'
+# oc login
 ----
++
+If your configuration is correct, you are logged in without entering separate
+credentials.
```
Comment (stuartchuan) [inline install_config/configuring_authentication.adoc]: There should add a different scenario, when no kerberos ticket issued or no krb5 workstation configured on the workstation, should fall back to basic authentication.

Comment (kalexand-rh) [reply install_config/configuring_authentication.adoc]: Ok. @enj, do you have any objections to the new commit?

**Comment by kalexand-rh** [comment]
@enj, @stuartchuan, is this one good to merge?

**Comment by stuartchuan** [comment]
The changes are OK for me now.

**Pull request merged**